tests/electoral_data_test.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+require_relative './test_helper'
+require_relative '../lib/member_page.rb'
+
+describe MemberPage do
+  around { |test| VCR.use_cassette('MemberPage', &test) }
+
+  subject do
+    url = 'http://www.parlament.mt/vella-george'
+    MemberPage.new(response: Scraped::Request.new(url: url).response)
+  end
+
+  describe 'electoral history' do
+    it 'should be eight terms long' do
+      subject.electoral_history.count.must_equal 8
+    end
+  end
+end
+
+describe 'Joe Cassar' do
+  around { |test| VCR.use_cassette('JoeCassar', &test) }
+
+  subject do
+    url = 'http://www.parlament.mt/joe-cassar'
+    MemberPage.new(response: Scraped::Request.new(url: url).response)
+  end
+
+  describe 'electoral history' do
+    it 'should have an end date' do
+      require 'pry'
+      subject.electoral_history['Twelfth Parliament']['Resignation from Parliament'].must_equal '3.11.15'
```

Comment:
I thought one of the goals here would have been to have this equal '2015-11-03'?